Assets/Scripts/Game/DaggerfallUI.cs

```diff
@@ -622,7 +622,7 @@ public AudioClip GetAudioClip(SoundClips clip)
         public void PlayOneShot(AudioClip clip)
         {
             if (audioSource)
-                audioSource.PlayOneShot(clip);
+                audioSource.PlayOneShot(clip, DaggerfallUnity.Settings.SoundVolume);
```

Comment:
There's no need to multiply the volume in all these places - everything goes through DaggerfallAudioSource anyway.

Please try my earlier suggestion to set volume in DaggerfallAudioSource right after spatialBlend. It should only be 2x lines of code and should cover everything.